torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16473,6 +16473,11 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
         torch_opinfo_name="isnan",
         supports_out=True,
     ),
+    ElementwiseUnaryPythonRefInfo(
+        "_refs.isreal",
+        torch_opinfo_name="isreal",
+        supports_out=True,
```

Comment:
It would likely need `supports_nvfuser=False,` because `ones_like` is not supported yet.

Reply:
Thanks! I also need to find out why there is a test failure that it is not decomposed...

Reply:
Found it.